model/registry/android.yaml

```diff
@@ -13,3 +13,37 @@ groups:
           (`os.version`) of the android operating system. More information can be found
           [here](https://developer.android.com/guide/topics/manifest/uses-sdk-element#ApiLevels).
         examples: ['33', '32']
+  - id: registry.android.lifecycle.events
```

Comment:
nit: do you think it's worth having a dedicated attributes group for it? Let's maybe merge it into the existing attributes group?

Reply:
I don't have any strong opinion also I don't know how "solid" the impl of tags is, but for the registry I have a preference for having different ids for the different things under the top level namespace. If we have everything in the same, it's often hard to find the 'group' that I'm interested, while like this it's very easy. Also, with tags you have to duplicate the tag on each attribute. this way not.

Reply:
the yaml structure so far is considered to be impl detail and we didn't try to enforce anything at all except the registry

Reply:
ok, having multiple ids makes it easier indeed to differentiate inside yaml file. Also good point about tag repetition. 
If Alex is ok with having multiple ids I will update existing registry entries to have the same format then

Reply:
I created an issue for this: https://github.com/open-telemetry/semantic-conventions/issues/887. Let's continue the discussion there and decide. Then we can come and make it consistent in all files (however we decide).